packages/landing/src/components/debug.tsx

```diff
@@ -34,7 +43,75 @@ export class Debug extends Component<{ map: maplibre.Map }> {
     );
   }
 
-  adjustOSM = (e: Event): void => {
+  _styleJson: Promise<Style> | null = null;
+  get styleJson(): Promise<Style> {
+    if (this._styleJson == null) {
+      this._styleJson = fetch(
+        WindowUrl.toTileUrl(MapOptionType.TileVectorStyle, Config.map.tileMatrix, 'topographic', 'topographic'),
```

Comment:
`Config.map.tileMatrix` probably not working for NZTM tileMatrix, do we want to disable the slider for NZTM?

Reply:
We don't disable OSM, we could make another PR to fix that.

Reply:
Sorry, I mean we can just disable the Topographic one for NZTM.
` <div className="debug__info">
          <label className="debug__label">Topographic</label>
          {debugSlider(this.adjustTopographic)}
        </div>`